package.json

```diff
@@ -20,7 +20,7 @@
     "prettier": "beemo prettier \"./packages/*/{src,test,storybook}/**/*.{js,jsx,ts,tsx,json,md}\"",
     "release": "yarn run prepare-release && lerna publish && yarn run postrelease",
     "test": "yarn run type && yarn run jest",
-    "test:watch": "beemo create-config jest --react && jest --watch"
+    "test:watch": "yarn run lint:fix && beemo create-config jest --react && jest --watch"
```

Comment:
Good idea to fix up lint errors while tests are running. Shall we add this to the `test` task also?

Reply:
I thought about that but decided not to because it will allow code that does not pass linting to pass CI when the CI runs `yarn test`. 

`yarn test:watch` on the other hand is only run by developer locally so that should be ok.

Reply:
agree 👍